tomcat/assets/monitors/thread_busy.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "[Tomcat] % of busy threads is high for host: {{host.name}}",
+  "type": "query alert",
+  "query": "avg(last_5m):( avg:tomcat.threads.busy{*} by {host} / max:tomcat.threads.max{*} by {host} ) * 100 > 70",
+  "message": "{{#is_alert}}\n\nALERT: The current amount of busy threads represents {{value}} % of the maximum number of allowed worker threads for host: {{host.name}}\n\n{{/is_alert}} \n\n{{#is_warning}}\n\nWARNING: The current amount of busy threads represents {{value}} % of the maximum number of allowed worker threads for host: {{host.name}}\n\n{{/is_warning}} \n\n",
```

Comment:
```suggestion
  "message": "{{#is_alert}}\n\nALERT: The current amount of busy threads represents {{value}} % of the maximum number of allowed worker threads for host: {{host.name}}\n\n{{/is_alert}} \n\n{{#is_warning}}\n\nWARNING: The current amount of busy threads represents {{value}} % of the maximum number of allowed worker threads for host: {{host.name}}\n\n{{/is_warning}}. \n\n",
```

Reply:
I avoided putting a dot at the end to avoid appending it to the hostname inlined (as there might be dots in the hostname).